app/src/features/projects/create/CreateProjectPage.tsx

```diff
@@ -337,7 +338,9 @@ const CreateProjectPage: React.FC = () => {
         component={{
           element: <ProjectDraftForm />,
           initialValues: {
-            draft_name: '' // TODO
+            draft_name: formikRef.current
```

Comment:
Would formikRef.current.values.project.project_name be more appropriate as the initial if statement?

Reply:
`formikRef.current` is possibly null, but if it's not null, then `formikRef.current.project` should be defined. But if we write `formikRef.current.values.project.project_name ?` then ESlint will still complain that "`forkmikRef.current` is possibly undefined."

Reply:
```
draft_name: formikRef?.current.values.project.project_name || ProjectDraftFormInitialValues.draft_name
```